app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -3,9 +3,18 @@ module Openshift
     class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+      KUBERNETES_EMS_TYPE = ManageIQ::Providers::Kubernetes::ContainerManager.ems_type
```

Comment:
@simon3z I am against adding something until there is an actual need for it.  If you want, it is probably more efficient to discuss face to face.